python/requirements-wheel-build.txt

```diff
@@ -1,5 +1,6 @@
 cython>=0.29.11
-numpy==1.16.6; python_version < "3.9"
-numpy==1.19.4; python_version >= "3.9"
+numpy==1.16.6; python_version < "3.9" and platform_machine != "aarch64"
+numpy==1.19.4; python_version >= "3.9" or platform_machine == "aarch64"
```

Comment:
There is no wheel available on aarch64 for earlier versions. We could compile numpy from source, but I'm not sure whether it worth it.